bin/qserv-check-integration.py

```diff
@@ -68,10 +68,12 @@ def _parse_args():
                        default="01",
                        help="Test case number")
     mode_option_values = ['mysql', 'qserv', 'all']
-    group.add_argument(
-        "-m", "--mode", dest="mode", choices=mode_option_values,
-        default='all',
-        help="Qserv test modes (direct mysql connection, or via qserv)")
+    group.add_argument("-m", "--mode", dest="mode", choices=mode_option_values,
+                       default='all',
+                       help="Qserv test modes (direct mysql connection, or via qserv)")
+    group.add_argument("-mn", "--multi", action="store_true", dest="multi_node",
```

Comment:
`-mn` option may be confusing, usually short option are one-letter, and `-ab` is usually interpreted as `-a -b`. I'd change it to one-letter (e.g. `-u`)